Urllib.hpp

```diff
@@ -1,6 +1,9 @@
 #include<locale>
 #include<map>
 #include<sstream>
+#include<algorithm>        
```

Comment:
Only algorithm is required XD

Reply:
Where are the test cases? lol

Reply:
Wait macha i am adding travis.yml and test cases